pkg/cmd/tests/options.go

```diff
@@ -164,6 +191,41 @@ func (o *TestFrameworkOptions) Validate(args []string) error {
 		errors = append(errors, fmt.Errorf("gcs-service-account-key-path and s3-credentials-file-path can't be set simultanously"))
 	}
 
+	if !tagWithOptionalDigestRegexp.MatchString(o.ScyllaDBVersion) {
+		errors = append(errors, fmt.Errorf(
+			"invalid scylladb-version format: %q. Expected format: <tag>[@<digest>], e.g., '6.2.0' or '6.2.0@sha256:abcdef...'",
```

Comment:
nit: imo `<tag>[@<digest>]` is sufficient, but if you prefer to leave the example it's ok